Buildings/Fluid/Humidifiers/Validation/DXDehumidifier.mo

```diff
@@ -23,289 +22,195 @@ model DXDehumidifier
     final mAir_flow_nominal=m_flow_nominal,
     final dp_nominal=100,
     final eneFac_nominal=3.412,
-    final per=per)
+    final per=per,
+    senTIn(T_start=302.05),
```

Comment:
This line and following line, is it possible to propagate the `sentinel.T_start`, `deHum.T_start` and `deHum.X_start` to be advanced parameters?